planet/api/experimental/basemaps_client.py

```diff
@@ -71,3 +81,23 @@ def list_quads_in_mosaic(
             for s in quad_list:
                 yield s
             url = resp.json().get('_links', {}).get('_next')
+
+    def get_quads_in_mosaic_for_region(self, mosaic_id, aoi):
```

Comment:
a docstring would be helpful here - it's not clear to me what format aoi needs to be in